src/docs/src/documentation/content/xdocs/zookeeperProgrammers.xml

```diff
@@ -841,9 +841,9 @@
     itself, ZooKeeper associates all the ids that correspond to a
     client with the clients connection. These ids are checked against
     the ACLs of znodes when a clients tries to access a node. ACLs are
-    made up of pairs of <emphasis>(scheme:expression,
+    made up of pairs of <emphasis>(scheme:id,
```

Comment:
> I think continuing to refer to the second term as an "expression" and explaining exactly what an "expression" is may be clearer.

Ok, I can do that instead. I was trying to consolidate the use of "id" and "expression" in these different contexts to prevent more confusion (as you realized). Will try to come up with something better :)